sky/backends/backend_utils.py

```diff
@@ -194,7 +194,12 @@ def fill_template(template_name: str,
         assert dst_parent_dir, f'Found relative destination path: {dst}'
 
         mkdir_parent = f'mkdir -p {dst_parent_dir}'
-        mv = (f'cp -r {_REMOTE_RUNTIME_FILES_DIR}/{src_basename} '
+        if os.path.isdir(os.path.expanduser(src)):
```

Comment:
This is a fix for: `sky launch` again will create a subfolder under the existing dst, instead of overwriting the contents. PTAL.

Reply:
`rsync -az` seems slower than `cp -r`, which I observed when working on the two recent PRs.

I got 2min36s and  2min29s on this new change, which seems slower than before. Can we use `cp`?

Reply:
Good point! Swtiched back to `cp -r`. Thanks!